ROMFS/px4fmu_common/init.d-posix/17001_TF-G1

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+#
+# @name ThunderFly TF-G1
+# ThunderFly TF-G1 autogyro airframe. Only for FlightGear simulator
+#
+# @type Autogyro
+# @class Autogyro
+#
+# @url https://github.com/ThunderFly-aerospace/TF-G1/
```

Comment:
I understand the model name is in captial letters, but adding this as a make target breaks the convention. No other model has a capital letter in the target name. For example, if750a is supposed to be IF750A but it is written in small case letters. Would it be possible to change this?

Reply:
renamed to tf-g1 in last commits